package.json

```diff
@@ -1,111 +1,24 @@
 {
-  "name": "qunit-dom",
-  "version": "2.0.0",
-  "description": "High Level DOM Assertions for QUnit",
-  "keywords": [
-    "browser",
-    "dom",
-    "ember-addon",
-    "html",
-    "qunit",
-    "qunit-plugin",
-    "testing"
-  ],
-  "repository": "https://github.com/simplabs/qunit-dom",
-  "license": "MIT",
-  "author": "simplabs GmbH",
-  "types": "dist/qunit-dom.d.ts",
-  "directories": {
-    "doc": "doc",
-    "test": "tests"
-  },
+  "private": true,
   "scripts": {
-    "build": "rollup -c",
-    "changelog": "lerna-changelog",
-    "docs": "npm run build && documentation build dist/qunit-dom.js --config documentation.yml -f md -o API.md",
-    "lint": "eslint . --cache",
-    "prepublish": "rollup -c",
-    "release": "release-it",
-    "test": "jest",
-    "test:coverage": "jest --coverage",
-    "test:ember": "ember test",
-    "test:watch": "jest --watchAll --notify"
-  },
-  "jest": {
-    "testEnvironment": "jsdom",
-    "preset": "ts-jest",
-    "testPathIgnorePatterns": [
-      "/node_modules/",
-      "/dist/"
-    ]
-  },
-  "dependencies": {
-    "broccoli-funnel": "^3.0.8",
-    "broccoli-merge-trees": "^4.2.0",
-    "ember-cli-babel": "^7.26.10",
-    "ember-cli-version-checker": "^5.1.2"
+    "docs": "npm run build && npm exec documentation build qunit-dom/dist/qunit-dom.js --config documentation.yml -f md -o API.md",
+    "build": "cd qunit-dom && npm run build",
+    "release": "cd qunit-dom && npm run release",
+    "lint": "pnpm --filter '*' lint",
```

Comment:
`pnpm --filter '*'` -> `pnpm -r` should have the same effect.
It doesn't matter though, wouldn't change that.